packages/wrapped-keys-lit-actions/src/lib/solana/internal/signMessage.js

```diff
@@ -0,0 +1,53 @@
+import { Keypair } from '@solana/web3.js';
+import bs58 from 'bs58';
```

Comment:
We can use ethers which is already in the nodes deno env instead of this

Reply:
What's more, we are dropping `bs58` from the SDK in v7 #662

Reply:
As long as it's the same functionality and returns the Solana message in base58 it's fine @MaximusHaximus